RIS/astm-uam-psu-ris-0.0.0.yaml

```diff
@@ -0,0 +1,570 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+      security:
+      - Authority:
+        - utm.strategic_coordination
+
+
+  # /ris/v1/resource_details/{entityid}/status:
+  # /ris/v1/resource_details/{entityid}/definition:
+
+components:
+  schemas:
+    ResourceShape:
+      description: The physical shape of the resource.  This is not the same as the resource extent saved in the DSS. 
+      anyOf:
+        - $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Circle'
+        - $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/LatLngPoint'
+        - $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Polygon'
+    ResourceDefinition:
+      required:
+        - name
+        - shape
+        - type
+      properties:
+        name:
+            description: Name of the resource
+            example: CHIPR
+            $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/TextShortType'
+        shape:
+          $ref: '#/components/schemas/ResourceShape'
+        altitude_lower:
+          description: Lower altitude limit of the resource, if any. 
+          $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Altitude'
+        altitude_upper:
+          description: Upper altitude limit of the resource, if any. 
+          $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Altitude'
+        type:
+          $ref: '#/components/schemas/ResourceType'
+    ResourceStatus:
+      required: 
+        - status
+      properties:
+        status: 
+          type: string
+          description: The current status of the resource. 
+          enum: 
+          - ACTIVE
+          - INACTIVE
+    ResourceType:
+      required: 
+        - resource_type
+      properties:
+        resource_type: 
+          type: string
+          description: The type of resource.  
+          enum: 
+          - WAYPOINT
+          - VOLUME
+          # TODO: Do we want to keep this generic or more specific and include intersection, merge, etc. ?
+    CapacityLimitType:
+      required:
+        - capacity_type
+      type: object
+      properties:
+        capacity_type:
+          type: string
+          description: |
+            The type of capacity limitation which is to be met for the resource. 
+          externalDocs:
+            url: TBD
+          enum: 
+          - FLOW_RATE_CONSTANT
+          - FLOW_RATE_FIXED_BIN
+          - INSTANTANEOUS
+    CapacityPropertyType:
+      required:
+        - capacity_property
+        - capacity_value
+      type: object
+      properties:
+        capacity_property:
+          type: string
+          description: |
+            Property of the capacity check which must be met.
+            - RATE describes the number of seconds between crossings enforced. (seconds)
```

Comment:
Typically the units of rate are events/time interval, not time interval (which is the inverse). So I'd suggest making this "the acceptable number of crossing (or events) per time interval (events per hour).

Reply:
Just to make sure, if we wanted to define 5 crossings allowed per hour, the properties you expect to see would be 1) RATE of 5 2)DURATION of 3600 (1 hour in seconds)

Reply:
I think that makes sense. This allows you to specify different rates but based on different time durations, so its pretty flexible.

Reply:
I updated the description for the properties - please take a look and see if you think it is clear enough.  Thanks!